### PR TITLE
Remove deprecated constants

### PR DESCRIPTION
**Pull request by OzGav**
I might be way off target here but warnings in HA 2024.10 beta and the associated release notes point to here https://developers.home-assistant.io/blog/2024/09/23/constants-media-player-deprecation/

Looking at the HA code I have come up with this PR. I won't be upset if this is totally wrong! :-)

### CODE REVIEW

**Comment by OzGav** [comment]
OK I pasted this into my local instance and the integration started without errors so I think it is good but I don't know how to fix the ruff lint problem...?

File: custom_components/mass/media_browser.py [hunk 1]
```diff
@@ -28,14 +19,14 @@
 if TYPE_CHECKING:
     from music_assistant.client import MusicAssistantClient
 
-MEDIA_TYPE_RADIO = "radio"
+MediaType.RADIO = "radio"
```
Comment (marcelveldt) [inline custom_components/mass/media_browser.py:22]: this is not the way to add an another enum member, better keep the previous custom constant if radio doesnt exist in the HA MediaType enum

Comment (OzGav) [reply custom_components/mass/media_browser.py:22]: I think I tried that but lint said no….something said no….

**Comment by marcelveldt** [comment]
deprecated by https://github.com/music-assistant/hass-music-assistant/pull/3057

**Pull request closed without merging**